### PR TITLE
[WIP] Messagebox fix performance

### LINKED ISSUE #10583: Typing a message becomes slower the more lines the message contains
### Description:

When I type a message that spans several lines, Rocket.Chat becomes very sluggish.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.63.2
* Operating System: Debian stretch
* Deployment Method(snap/docker/tar/etc): tar
* Number of Running Instances: 1
* DB Replicaset Oplog: Disabled
* Node Version: v8.9.3
* mongoDB Version: mongodb-org 3.2.17

### Steps to Reproduce:

1. Type several lines into the message box (separate the lines with Shift+Enter). The lines don't have to be long - ten lines with a single character will trigger the issue just fine:

	```
	a
	a
	a
	a
	a
	a
	a
	a
	a
	```

	More lines seem to increase the sluggishness.

2. Now try to type a complete sentence, e.g. `The quick brown fox jumps over the lazy dog`. Notice how sluggish the typing has become.

I can reproduce this issue both in the Electron client on Ubuntu, and on the website.

### Expected behavior:

Typing shouldn't slow down just because of a few lines.

### Actual behavior:

Typing slows down noticeably.

### PR DESCRIPTION
**Pull request by ggazzo**
should close #10583

### CODE REVIEW

File: packages/rocketchat-ui/client/lib/textarea-autogrow.js [hunk 1]
```diff
@@ -6,22 +7,20 @@
 	 * http://github.com/jaz303/jquery-grab-bag/tree/master/javascripts/jquery.autogrow-textarea.js
 	 */
 	$.fn.autogrow = function(options) {
+		var shadow = $("div.autogrow-shadow");
```
Comment (rssilva) [inline packages/rocketchat-ui/client/lib/textarea-autogrow.js]: Can we use `let` here?

Comment (rssilva) [reply packages/rocketchat-ui/client/lib/textarea-autogrow.js]: Maybe we could add a `TODO` and investigate if there is a way to pass the container element to search this `.autogrow-shadow` class. Doing like this we are looking at the whole DOM.

**Comment by rssilva** [comment]
It seems that the `keypress` event was decreased to the half 🎉
 
current:

screen shot 2018-08-07 at 11 32 34 am

this branch:

screen shot 2018-08-07 at 11 32 51 am

File: packages/rocketchat-ui/client/lib/textarea-autogrow.js (hunk 1 above)
Comment (ggazzo) [reply packages/rocketchat-ui/client/lib/textarea-autogrow.js]: actually this code was a copy/paste thats why I'm not using >ES5 on code (just were I changed)

Comment (ggazzo) [reply packages/rocketchat-ui/client/lib/textarea-autogrow.js]: and about the find, we can use `body > .autogrow-shadow` or use an `id` instead, but this code just runs "one time"(one time per opened room) so I dont think will change a lot, but I will change

**Comment by engelgabriel** [comment]
Fixes https://github.com/RocketChat/Rocket.Chat.Electron/issues/773

**Pull request merged**